### PR TITLE
Add advent candle in december

### PR DESCRIPTION
**Pull request by hrjakobsen**
Inspired by #101 and a suggestion by a fember, this pull request adds an advent candle to stregsystemet in December. I'm not sure what the size of the candle should with the new screen and all, so I just picked something that looked alright on my 1920x1080 screen.

Candle on December 1st

Candle on December 12th

Candle on December 24th (evening)

### CODE REVIEW

**Comment by hrjakobsen** [comment]
I originally implemented the flame as a css transition, but removed it as I feared that the pi couldn't handle it. I can easily change the flame.

Something like this would be really cool: https://codepen.io/2xsamurai/pen/EKpYMg

Do you think the Pi can handle it?

**Comment by VirtualSatai** [comment]
billede

**Comment by hrjakobsen** [comment]
I changed the flame to the one in the codepen above.

File: stregsystem/static/stregsystem/stregsystem.css [hunk 1]
```diff
@@ -82,3 +82,35 @@ table h2, table h3 {
   max-height: 75%;
   max-width: 400px;
 }
+
+.flame-main {
+  /*animation-name: flameWobble;*/
```
Comment (hrjakobsen) [inline stregsystem/static/stregsystem/stregsystem.css]: If this line is uncommented, there will be a slow fire animation. I think the Pi can handle it, but it is disabled by default.

**Comment by DelusionalLogic** [comment]
It needs to be tested on a PI, but otherwise I think it's pretty nice.

We should also consider how this will look on other screens and other aspect ratios. I guess you could always ```display: none``` in a mediatag if it's a problem.

**Pull request merged**